scoper.inc.php

```diff
@@ -37,6 +37,13 @@
     'whitelist' => [
         \Composer\Autoload\ClassLoader::class,
         'Safe\*',
+        // PHP 8.0
+        'T_NAME_QUALIFIED',
+        'T_NAME_FULLY_QUALIFIED',
+        'T_NAME_RELATIVE',
+        'T_MATCH',
+        'T_NULLSAFE_OBJECT_OPERATOR',
+        'T_ATTRIBUTE',
```

Comment:
Taken from here: https://github.com/nikic/PHP-Parser/blob/1d1bc8a36459d67acef508b952dad097960b110e/lib/PhpParser/Lexer.php#L413

Reply:
>Could this be added to https://github.com/humbug/php-scoper/blob/master/src/Reflector.php#L64?

Reply:
Absolutely. I thought there must be something like this, but couldn't find this straight away.